docker/federation-test/run-tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -e
+
+pushd ../../server/
+cargo build
+popd
+
+sudo docker build ../../ --file ../federation/Dockerfile --tag lemmy-federation:latest
+
+sudo docker-compose --file ../federation/docker-compose.yml --project-directory . up -d
+
+# TODO: need to wait until the instances are initialised
```

Comment:
Oops I left that comment, maybe you can remove it.

Reply:
Done.